ReadMe.md

```diff
@@ -1,3 +1,34 @@
-# MAGE SDK for JavaScript (ES5)
+# MAGE SDK (Web)
 
-This is a work-in-progress.
+This is the official [MAGE](https://www.npmjs.com/package/mage) SDK for web browsers.
+
+- Completely written in ES5 to be compatible with the widest range of browsers.
+- Uses CommonJS (`require()` calls and `module.exports`).
+
+## Compatibility
+
+This version of `mage-sdk-js` works with **MAGE >=1.1.0**.
+
+## Installation
+
+In the example below, we assume you use `NPM` to install dependencies for your web application.
```

Comment:
I don't believe we need to:

  1. Implicitly, we are not supporting anything thats not documented
  2. I could see someone make it work with Bower + requirejs or some other combination of tools at some point anyway